apps/fluent-tester/src/FluentTester/TestComponents/ButtonExperimental/ButtonIconTestSection.tsx

```diff
@@ -9,7 +9,7 @@ export const ButtonIconTest: React.FunctionComponent = () => {
   const fontBuiltInProps = {
     fontFamily: 'Arial',
     codepoint: 0x2663,
-    fontSize: 32,
+    fontSize: 24,
```

Comment:
This font size gives us approximately the size of icon we'd want for small/medium buttons